resources/dicts/events/lifegen_events/murder.json

```diff
@@ -347,7 +351,8 @@
         "Day after day, v_c has been getting under your fur. They're constantly undermining you, constantly finding ways to make you look bad in front of the leader. Eventually, you see what v_c is doing. They want to get you demoted. They want your job. Being deputy is something you've always valued, and you won't let v_c cheat you out of the life you've earned, the role you've fought so hard for. You suggest to them that you two should talk after such tense moments between you two. It's dark when you're both out, every other cat asleep. As v_c turns to stare up at the stars, you unsheathed your claws and successfully preserve your role."
     ],
     "murder deputy medicinecat": [
-        "You know that v_c would never accept you as leader, maybe even enough to bar you from your deserved nine lives. As your leader's lives tick down further and further, you decide to take matters into your own paws, and push them into the river to be lost in the dark water. Their replacement will be much more understanding."
+        "You know that v_c would never accept you as leader, maybe even enough to bar you from your deserved nine lives. As your leader's lives tick down further and further, you decide to take matters into your own paws, and push them into the river to be lost in the dark water. Their replacement will be much more understanding.",
```

Comment:
can probably be reworded

"You know full-well that v_c would never accept you as leader. They might even bar you from receiving your well-deserved nine lives. You decide to take matters into your own paws before it's too late, and before long v_c floats lifeless down the river. Surely, their replacement will be more understanding of you."